fre/cmor/README.md

```diff
@@ -29,18 +29,39 @@ this subtool's help, and command-specific `run` help:
-
-    Options:
-      -d, --indir TEXT         Input directory  [required]
-      -l, --varlist TEXT       Variable list  [required]
-      -r, --table_config TEXT  Table configuration  [required]
-      -p, --exp_config TEXT    Experiment configuration  [required]
-      -o, --outdir TEXT        Output directory  [required]
-      --help                   Show this message and exit.
+> fre cmor run --help                        
+Usage: fre cmor run [OPTIONS]                                                 
+                                                                              
+  Rewrite climate model output files with CMIP-compliant metadata for down-   
+  stream publishing                                                           
+                                                                              
+Options:                                                                      
+  -d, --indir TEXT         directory containing netCDF files. keys specified  
+                           in json_var_list are local variable names used for 
+                           targeting specific files in this directory         
+                           [required]                                         
+  -l, --varlist TEXT       path pointing to a json file containing directory  
+                           of key/value pairs. the keys are the 'local' names 
+                           used in the filename, and the values pointed to by 
+                           those keys are strings representing the name of the
+                           variable contained in targeted files. the key and  
+                           value are often the same, but it is not required.  
+                           [required]                                         
+  -r, --table_config TEXT  json file containing CMIP-compliant per-           
+                           variable/metadata for specific MIP table. The MIP  
+                           table can generally be identified by the specific  
+                           filename (e.g. 'Omon')  [required]                 
+  -p, --exp_config TEXT    json file containing metadata dictionary for       
+                           CMORization. this metadata is effectively appended 
+                           to the final output file's header  [required]      
+  -o, --outdir TEXT        directory root that will contain the full output   
+                           and output directory structure generated by the    
+                           cmor module upon request.  [required]              
+  -v, --opt_var_name TEXT  optional, specify a variable name to specifically  
+                           process only filenames matching that variable name.
+                           I.e., this string help target local_vars, not      
+                           target_vars.                                       
+  --help                   Show this message and exit.                        
```

Comment:
I can't ever really disagree with documentation, and don't disagree with this, but we'll have to keep this up-to-date with the actual --help output content which is defined elsewhere. Eventually, documentation like this will be auto-created through sphinx somehow.

Reply:
I've been pondering this too. hear me out for a second- I know that the way we generate coverage badges in our pipeline is a bit hacky- utilizing the deployed github pages to host the generated image at a static URL for linking to the README. 

But... there DO exist workflows that edit the coverage badge in the repo directly and makes the commit itself- i tried to make those work but couldn't figure it out on a reasonable timeline.

Why do i bring this up? Well, in theory if a workflow could commit to the repo for a generated test-coverage badge, it should also be able to commit a quick README edit based on the latest `fre <tool> --help` or `fre <tool> <subtool> --help`